STARTERKIT/components/components/slider/slider.js

```diff
@@ -0,0 +1,26 @@
+(function (Drupal, $, debounce, drupalSettings) {
+  console.log('bu');
+  Drupal.behaviors.slider = {
+    attach: function (context, settings) {
+      var opt = [
+        {
+          slider: $('.slick-slider .field__items'),
+          options: {
+            dots: true,
+            infinite: false,
+            slidesToScroll: 1,
+            slidesToShow: 1,
+            speed: 300
+          }
+        }
+      ];
+
+      for (var i = 0; i < opt.length; i++) {
```

Comment:
Not sure about this way of initializing the sliders, they will all share the same config/options. Maybe better to provide a `slider.js` as example but commented out / not included.